### PR TITLE
Allow using https in acceptance tests Activiti/Activiti#2406

### PR DESCRIPTION
**Pull request by mteodori**

### CODE REVIEW

File: runtime-acceptance-tests/src/main/resources/config-env.properties [hunk 1]
```diff
@@ -1,7 +1,9 @@
...
-graphql.ws.url=ws://${GATEWAY_HOST}/ws/graphql
-graphql.url=http://${GATEWAY_HOST}/graphql
\ No newline at end of file
+sso.url=${sso.protocol:http}://${sso.host}/auth
+auth.url=${sso.url}/realms/${realm}/protocol/openid-connect/token
+gateway.url=${gateway.protocol:http}://${gateway.host}
+audit.event.url=${gateway.url}/audit
+runtime.bundle.url=${gateway.url}/rb-my-app
+query.url=${gateway.url}/query
+graphql.ws.url=#{"${gateway.url}".replace('http', 'ws')}/ws/graphql
```
Comment (igdianov) [inline runtime-acceptance-tests/src/main/resources/config-env.properties]: @mteodori For `https` in the GATEWAY_HOST, the graphql.ws.url should use `wss:` protocol scheme..

Comment (mteodori) [reply runtime-acceptance-tests/src/main/resources/config-env.properties]: that is what the replacement is doing, I think I should just change to replaceFirst

Comment (mteodori) [reply runtime-acceptance-tests/src/main/resources/config-env.properties]: all good with https disabling the non working tests and security policies

Comment (igdianov) [reply runtime-acceptance-tests/src/main/resources/config-env.properties]: @mteodori I have HTTPS in my environment, I will try deploy and run acceptance tests there with your change and then check notes with you.

**Pull request merged**